test/ssc_test/cmod_utilityrate5_test.cpp

```diff
@@ -214,14 +214,10 @@ TEST(cmod_utilityrate5_eqns, Test_Residential_net_metering_credits_in_may) {
     EXPECT_NEAR(36.6, cost_with_system, 0.1);
 
     int length;
-    ssc_number_t* excess_dollars = ssc_data_get_array(data, "year1_excess_dollars_earned", &length);
+    ssc_number_t* excess_dollars = ssc_data_get_array(data, "year1_nm_dollars_applied", &length);
     float may_dollars = excess_dollars[credit_month];
```

Comment:
Can you please do a sanity check with a credit_month of 0 and another test with the credit_month 12 or higher?

Reply:
Great catch! 0 had some issues which were fixed in the latest push. 12 was handled by the vartable input checks.